__tests__/app.test.js

```diff
@@ -325,3 +325,81 @@ describe("6. 09-GET_api_articles_article.id_comments /api/articles/:article_id/c
       });
   });
 });
+
+describe("8. 10-POST_api_articles_article.id_comments /api/articles/:article_id/comments", () => {
+  test("status:201, responds with an array of new comments for given test article_id objects", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "I love pizza",
+    };
+    return request(app)
+      .post(`/api/articles/2/comments`)
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toBeInstanceOf(Array);
+        expect(body.comment).toHaveLength(1);
+        body.comment.forEach((testcomment) => {
+          expect(testcomment).toEqual(
+            expect.objectContaining({
+              comment_id: expect.any(Number),
+              body: expect.any(String),
+              article_id: expect.any(Number),
+              author: expect.any(String),
+              votes: expect.any(Number),
+              created_at: expect.any(String),

```

Comment:
i would make this test stricter by expecting an object that is the comment you sent including the added columns! 

body = "i love pizza"
article_id = 2

etc. etc. 😄